Composer/packages/lib/bot-deploy/src/botProjectDeploy.ts

```diff
@@ -89,6 +90,22 @@ export class BotProjectDeploy {
       value: scope,
     };
   }
+  private async getTenantId() {
+    if (!this.accessToken) {
+      throw new Error('Invalid Access Token');
+    }
+    try {
+      const tenantUrl = `https://management.azure.com/tenants?api-version=2020-01-01`;
+      const options = {
+        headers: { Authorization: `Bearer ${this.accessToken}` },
+      } as rp.RequestPromiseOptions;
+      const response = await rp.get(tenantUrl, options);
```

Comment:
Optional: It would be cool to have a comment with a link to the documentation of the api you are calling so future maintainers of this code can quickly see the shape and input / output of the api.